src/pages/hardware/samples/python.tsx

```diff
@@ -0,0 +1,212 @@
+import { ReactNode } from "react";
+import { Page } from "../../../components";
+import { PageTitleProps } from "../../../components/PageTitle/PageTitle";
+import { HeaderSections } from "../../../components/Header/headerSectionsEnum";
+import Accordion from "../../../components/Accordion/Accordion";
+import { TableOfContentsItem } from "../../../components/TableOfContents/TableOfContents";
+import { CodeSample } from "../../../components/CodeSamplesContainer";
+
+const spoofAnExternalDevice: ReactNode = (
+  <div className="paragraph">
+    <p>
+      Run this python script while connected to a GO device via an IOX-RS232
+    </p>
+    <CodeSample
+      language="python"
+      code={
+`import serial
+import time
+      
+def checksum(message):
```

Comment:
please use tab (or 4 spaces) indenting, not one space. I know that's what they have on the current site but it's silly